processor/redactionprocessor/config.go

```diff
@@ -23,6 +23,11 @@ type Config struct {
 	// without being changed or removed.
 	IgnoredKeys []string `mapstructure:"ignored_keys"`
 
+	// RedactAllTypes is a flag to allow the processor to redact all span
+	// attribute values including those that are not strings. By default
+	// it is false and only string values are redacted.
```

Comment:
This applies to all attributes, not just span attributes
```suggestion
	// RedactAllTypes of attributes, including those that are not string.
	// By default only string values are redacted.
```